test/command_line/helper/command_runner.rb

```diff
@@ -105,17 +105,20 @@ def grndb(command, *arguments)
   def find_program(name, options={})
```

Comment:
`name = name + RbConfig::CONFIG["EXEEXT"]` here may be better.

Reply:
I've fixed it.